tests/lib/rules/no-decorators-before-export.js

```diff
@@ -0,0 +1,70 @@
+const rule = require('../../../lib/rules/no-decorators-before-export');
+const RuleTester = require('eslint').RuleTester;
+
+const eslintTester = new RuleTester({
+  parser: require.resolve('@babel/eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2022,
+    sourceType: 'module',
+    requireConfigFile: false,
+    babelOptions: {
+      babelrc: false,
+      plugins: [['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true }]],
+    },
+  },
+});
+
+eslintTester.run('no-decorators-before-export', rule, {
+  valid: [
+    'export default class {}',
+    'export default class Foo {}',
+    'class Foo {}\nexport default Foo;',
+    'export class Foo {}',
+    'class Foo {}\nexport { Foo };',
+    "import classic from 'ember-classic-decorators';\n@classic\nclass Foo {}\nexport { Foo };",
+    "import classic from 'ember-classic-decorators';\n@classic\nclass Foo {}\nexport default Foo;",
+  ],
+  invalid: [
+    // basic
+    {
+      code: "import classic from 'ember-classic-decorators';\n@classic\nexport class Foo {}",
```

Comment:
I'm finding these single-line strings with \n a hard to read. Normally, we use multi-line template strings.